src/utils/flags.ts

```diff
@@ -0,0 +1,8 @@
+import APIError from '@/app/api/core/exceptions/APIError'
+import httpStatus from 'http-status'
+
+export const checkIfFeatureFlagged = (flag: boolean) => {
+  if (flag) return
+
+  throw new APIError(httpStatus.NOT_FOUND, 'The request resource does not exist')
```

Comment:
I think the check runs in webhooks right? If we throw error in webhooks, will it retry? even if it dont, it will show as error log..

We should return 200 in case of webhooks.. that way we are only bypassing the logic.